tiledb/sm/crypto/CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ commence(object_library tiledb_crypto)
         this_target_link_libraries(bcrypt)
     else()
         find_package(OpenSSL_EP REQUIRED)
-	#this_target_link_libraries(OpenSSL::Crypto)
-	target_link_libraries(tiledb_crypto PRIVATE OpenSSL::Crypto)
+	    #this_target_link_libraries(OpenSSL::Crypto)
+	    target_link_libraries(tiledb_crypto PRIVATE OpenSSL::Crypto)
```

Comment:
If you're not going to use `this_target_link_libraries`, you need documentation explaining why.

Nit: indentation

Reply:
Fixed indentation. I don't know why we didn't use `this_target_link_libraries`, let me investigate…